src/routes/add-article.ts

```diff
@@ -28,7 +28,7 @@ export default (): AppMiddleware => (
       throw new createHttpError.BadRequest(`Article must have at least one ${termToString(schema('name'))}`);
     }
 
-    const newId = blankNode(uniqueString());
+    const newId = namedNode(uniqueString());
```

Comment:
The documentation says [this should be an IRI](http://rdf.js.org/data-model-spec/#namednode-interface), but we have [good reasons to use host-free values](
https://github.com/libero/publisher/issues/367#issuecomment-574638754).